blog/content/posts/2025-09-05-php-sdk.md

```diff
@@ -0,0 +1,29 @@
++++
+date = '2025-09-05T00:00:00Z'
+title = 'Announcing the Official PHP SDK for MCP'
+author = 'David Soria Parra (Lead Maintainer), Christopher Hertel (Symfony), Roman Pronskiy (PHP Foundation)'
+tags = ['announcement', 'community']
++++
+
+The official [PHP SDK](https://github.com/modelcontextprotocol/php-sdk) for the Model Context Protocol is now generally available. 
+
+Built in collaboration with the [PHP Foundation](https://thephp.foundation/) and [Symfony](https://symfony.com/), the PHP SDK handles protocol details, so developers don’t have to worry about low-level mechanics and can focus on building their applications. 
+
+The initial release enables PHP developers to build MCP [servers](https://modelcontextprotocol.io/docs/learn/server-concepts), exposing [tools](https://modelcontextprotocol.io/docs/learn/server-concepts#tools-ai-actions), [prompts](https://modelcontextprotocol.io/docs/learn/server-concepts#prompts-interaction-templates), and [resources](https://modelcontextprotocol.io/docs/learn/server-concepts#resources-context-data) to AI applications. Support for PHP applications to act as MCP [clients](https://modelcontextprotocol.io/docs/learn/client-concepts) will follow. 
+
+The PHP SDK now joins 9 other [officially supported language SDKs](https://modelcontextprotocol.io/docs/sdk) in the MCP ecosystem, making it easier for developers everywhere to adopt MCP in their preferred language.
+
+## Get involved
+
+The PHP SDK is now open to the community to install, test, and contribute: 
+
+- SDK repo: [modelcontextprotocol/php-sdk](https://github.com/modelcontextprotocol/php-sdk)  
+- Composer package: [mcp/sdk](https://packagist.org/packages/mcp/sdk) 
+
+We welcome your feedback and contribution, including issues, documentation improvements, and pull requests. Framework-specific integrations and real-world examples are also particularly valuable.
+
+## Thanks to the MCP community 
+
+This release consolidates earlier community work into a single, trusted implementation. The SDK is maintained by the [Symfony](https://symfony.com/) team, with [Kyrian Obikwelu](https://github.com/CodeWithKyrian) joining as a core maintainer based on his previous [PHP-MCP](https://github.com/php-mcp) work. The [PHP Foundation](https://thephp.foundation/) helped to coordinate the initiative with support from the members of MCP steering group.
```

Comment:
Just the governance semantics, but is "core maintainer" [mapping to the existing outline](https://modelcontextprotocol.io/community/governance#current-core-maintainers)?